Python prints a stack trace when your code throws an exception. A stack trace is often also referred to as a stack traceback, backtrace, or traceback. However, we prefer using the stack trace. Python prints a stack trace when your code throws an exception. A stack trace is often also referred to as a stack traceback, backtrace, or traceback. However, we prefer using the stack trace. In other words, a stack trace prints all the calls prior to the function that raised an exception. In all cases, the last line of a stack trace prints the most valuable information as here the error gets printed. All information above the thrown error consists of function calls that help you to trace the issue faster.